src/app/static/src/app/components/ResetConfirmation.vue

```diff
@@ -48,7 +53,9 @@
     interface Computed {
         changesToRelevantSteps: StepDescription[],
         currentVersionId: string | null,
-        errorsCount: number
+        errorsCount: number,
+        handleVersionNote: string,
+        currentVersionNote: string | null
```

Comment:
In what circumstance can `currentVersionNote` be null? The computed value either returns the current value if there is one, or the empty string.